getting-started/background-processing/TranslateWorker/Startup.cs

```diff
@@ -45,7 +45,7 @@ public void ConfigureServices(IServiceCollection services)
                 FirestoreDb.Create(GetFirestoreProjectId()));
             services.AddSingleton<Google.Cloud.Translation.V2.TranslationClient>(
                 Google.Cloud.Translation.V2.TranslationClient.Create());
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc(options => options.EnableEndpointRouting = false).SetCompatibilityVersion(CompatibilityVersion.Version_2_1);

```

Comment:
(And here, if you want to - I won't point out any more that come up later.)

Reply:
Yep, done here as well, no other places. Thansk!